.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: java
+install: true
+dist: trusty
+jdk: openjdk11
+before_script:
+  - cd repair-service/
```

Comment:
why do we need to open specific dir here?
Maybe it would be better to change project structure as well as build tool?